PSIPro.ino

```diff
@@ -292,8 +302,8 @@ unsigned long commandTiming = 0;
 int level[10] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
 
 //Serial Stuff
-int lastPSIeventCode     = defaultPattern;
-bool firstTimeCode      = true;
+int lastPSIeventCode  = defaultPattern;
```

Comment:
No need for 2 spaces here, reduce this to one after the lastPSIeventCode, and match the firstTimeCode.